README.md

```diff
@@ -70,6 +70,7 @@ Also creates the following files:
 
 * `tag` containing the git tag name of the release being fetched.
 * `version` containing the version determined by the git tag of the release being fetched.
+* `body` containing the body text of the release determined by the git tag of the release being fetched.
```

Comment:
this could just say "containing the body text of the release" - i'll just fix after merging